### DISCUSSION TITLE
Allow switching image layers on/off for the whole manifest

### DISCUSSION THREAD

**Discussion by lutzhelm** [ProjectMirador/mirador#3466] [General]
This is an idea for a somewhat advanced feature; most certainly not an issue which will be solved before a first stable M3 release.

In the manuscript portal project (Handschriftenportal), we have a feature request for allowing the user to switch between certain types of images not just for the current Canvas but for the whole manifest. For example, if every Canvas contains a daylight and a UV light photograph, a user wants to be able to navigate through the manifest with only UV images visible without having to enable / disable layers for each new Canvas.

For the moment, this is purely theoretical because there are probably no IIIF resources providing the necessary information to enable such a feature. I suggested a cookbook recipe IIIF/cookbook-recipes#163 to be able to convey this information. I'd be happy about any comments or suggestions on that.

On the Mirador end, I think this is primarily a UI problem for providing controls for both the Canvas and the Manifest, as well as the issues of parsing additional information about an AnnotationCollection, but I think we could use the layers state and reducers as they are now for this purpose.

**Comment by jvine** [comment]
Something similar came up in our initial scenarios for M3 - turning on a specific layer for all canvases, in the context of the gallery view. The panels in gallery view are eventually meant to provide a way to apply canvas-level features (annos, layers, ...?)  to multiple/all canvases - but we didn't have time to work this out in detail.
